.serena/memories/suggested_commands.md

```
@@ -0,0 +1,181 @@
+```bash
+# List KV namespaces
+npx wrangler kv:namespace list
+
+# Create KV namespace
+npx wrangler kv:namespace create "AKYO_KV"
+```
+
+## Useful URLs
+
+### Local Development
+- Gallery: http://localhost:3000/zukan
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)